src/pages/PostsManager.js

```diff
@@ -20,134 +20,33 @@ import { compose } from 'recompose';
 import PostEditor from '../components/PostEditor';
 import ErrorSnackbar from '../components/ErrorSnackbar';
 
-const styles = theme => ({
-  posts: {
-    marginTop: theme.spacing(2),
-  },
-  fab: {
-    position: 'absolute',
-    bottom: theme.spacing(3),
-    right: theme.spacing(3),
-    [theme.breakpoints.down('xs')]: {
-      bottom: theme.spacing(2),
-      right: theme.spacing(2),
-    },
-  },
-});
-
-const API = process.env.REACT_APP_API || 'http://localhost:3001';
-
-class PostsManager extends Component {
-  state = {
-    loading: true,
-    posts: [],
-    error: null,
-  };
-
-  componentDidMount() {
-    this.getPosts();
-  }
-
-  async fetch(method, endpoint, body) {
-    try {
-      const response = await fetch(`${API}${endpoint}`, {
-        method,
-        body: body && JSON.stringify(body),
-        headers: {
-          'content-type': 'application/json',
-          accept: 'application/json',
-          authorization: `Bearer ${await this.props.authService.getAccessToken()}`,
-        },
+import { init, AuthType, LiveboardEmbed, EmbedEvent, RuntimeFilterOp } from '@thoughtspot/visual-embed-sdk';
+
+export const PostsManager = ({userName}) => {
+  let { pinboardId } = useParams();
+	React.useEffect(() => {
+    if (pinboardId && userName) {
+      const liveboardEmbed = new LiveboardEmbed('#embed', {
+          frameParams: {
+              width: '100%',
+              height: '100%',
+          },
+          fullHeight: true,
+          liveboardId: pinboardId,
+          
+          runtimeFilters: [{
+            columnName: 'sales',
+            operator: RuntimeFilterOp.GE,
+            values: ['1000']
+        }]
```

Comment:
Add a date range widget on the UI, when the user inputs a date range. The runtime filter is applied to be within those dates.